app/src/main/res/layout/activity_description.xml

```diff
@@ -182,22 +183,23 @@
         android:id="@+id/activityGoDescription"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:fontFamily="@font/armata"
         android:layout_marginStart="20dp"
-        android:layout_marginTop="100dp"
+        android:layout_marginTop="20dp"
+        android:fontFamily="@font/armata"
         android:text="How to go"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/activity_description_description" />
 
     <Button
         android:id="@+id/activityChatDescription"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="100dp"
```

Comment:
Why this button is soooo BIG ?

Reply:
This is to avoid the error "androidx.test.espresso.PerformException: Error performing 'single click' on view". In addition, in case the button remains that big until our next meeting with the TAs, it will be the starting point of an interesting discussion about why we face issues with tests that systematically pass “./gradlew connectedCheck”, but not Cirrus CI 😉